src/components/thumbnail/my-work.tsx

```diff
@@ -21,6 +21,7 @@ export class MyWorkComponent extends BaseComponent<IProps, {}> {
     }
     return (
       <div className="my-work">
+        <div className="header">My Work</div>
```

Comment:
I stuck the headers in on the My Work and Class Logs tabs, but limited the UI changes in those sections to just that (other than things they inherit from the CSS).  There is obviously more in the UI spec that could be done, but this was the path of least resistance to make things in the other two sections reasonably match the UI changes in Class Work.